app/assets/stylesheets/frontend/views/attachments/_csv-preview.scss

```diff
@@ -0,0 +1,15 @@
+.csv-preview {
+  border: govuk-spacing(1) solid govuk-colour("light-grey");
```

Comment:
Do we need this? Taking a look at the gem for tables + the Design System I'm thinking this is something bespoke and therefore questioning if it's needed.

Reply:
We don't need it necessarily. I primarily left it there to keep it inline with the existing design. Here's how it looks if I remove it:
![Screenshot 2021-04-13 at 09 47 23](https://user-images.githubusercontent.com/64783893/114525352-f27abc00-9c3d-11eb-82af-66959034b522.png)
I personally prefer it being there as a little visual enhancement so my instinct is to keep it, but you're right that it's bespoke so it's a small risk.